core/Cargo.toml

```diff
@@ -10,13 +10,17 @@ license.workspace = true
 repository.workspace = true
 
 [dependencies]
+cfg-if = "1.0"
```

Comment:
nit: I think in this case we could avoid the dependency. It's possible to do the following. `cfg_if` is fine it's just an FYI.

```rust
pub const MAX_SIGNATURE_LENGTH: usize = if cfg!(feature = "larger-signature") {
    10_000_000
} else {
    1_000
};
```

Reply:
The `cfg-if` is required because your proposed solution requires any referenced crates within the `if` block to exist. Since I'm using it to conditionally reference the `pqcrypto_mldsa` crate, if I used your proposed approach, that crate will always have to be included as a dependency in order for it to compile.